### PR TITLE
Add: organization setting for time format

### PR DESCRIPTION
**Pull request by yusukegoto**
## What type of PR is this? (check all applicable)

- [x] Feature

## Description
Add small patch of time format. I left timezone support for some complex reasons.

## Related Tickets & Documents
https://github.com/getredash/redash/issues/618

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)

Settings

screen

Query

query

### CODE REVIEW

**Comment by rauchy** [comment]
Thanks! LGTM 👍

File: redash/handlers/authentication.py [hunk 1]
```diff
@@ -196,10 +196,13 @@ def base_href():
 def date_format_config():
```
Comment (rauchy) [inline redash/handlers/authentication.py]: Perhaps this could be renamed to `date_time_format_config`

Comment (yusukegoto) [reply redash/handlers/authentication.py]: fixed in https://github.com/getredash/redash/pull/3754/commits/09de0b7ef49da369844078f0d9601fc47387d828

**Comment by arikfr** [comment]
Can you please trigger a new build?

**Comment by yusukegoto** [comment]
@arikfr 
Could you help me? How can I build with `PERCY_TOKEN` s. 
Pushing commits to my repo seems useless.
https://circleci.com/gh/yusukegoto/redash/41?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

**Comment by gabrieldutra** [comment]
Hi @yusukegoto :slightly_smiling_face:

Do you have CircleCI configured for your `redash`'s fork? If so, I think it doesn't trigger in our side :confused: (therefore it doesn't get our token).

**Comment by yusukegoto** [comment]
Unit tests has passed! I created my own circleci settings for forked repo.

**Comment by yusukegoto** [comment]
Please review percy diff :bow:

**Comment by arikfr** [comment]
Thanks, @yusukegoto !

@gabrieldutra note that you will need to rebase this into your organization settings PR.

**Pull request merged**